sdk/storage/azure-storage-file-share/azure/storage/fileshare/aio/_file_client_async.py

```diff
@@ -215,9 +215,9 @@ def __init__(  # type: ignore
         :keyword dict(str,str) metadata:
             Name-value pairs associated with the file as metadata.
         :keyword lease:
-            Required if the file has an active lease. Value can be a ShareFileLeaseClient object
+            Required if the file has an active lease. Value can be a ShareLeaseClient object
             or the lease ID as a string.
-        :paramtype lease: ~azure.storage.fileshare.ShareFileLeaseClient or str
+        :paramtype lease: ~azure.storage.fileshare.ShareLeaseClient or str
```

Comment:
These docstring types should be `~azure.storage.fileshare.aio.ShareLeaseClient`

Reply:
good catch!